src/chain/chain_accumulator.rs

```diff
@@ -397,7 +468,7 @@ mod test {
     fn reset_all_completed(data: TestData) {
         let mut acc = ChainAccumulator::default();
         let _ = acc.add_proof(data.event.clone(), data.first_proof, data.signature.clone());
-        let _ = acc.poll_event(data.event.clone());
+        let _ = acc.poll_event(data.event.clone(), Default::default());
 
         let result = acc.reset_accumulator(&data.our_id);
```

Comment:
Not seeing the unti test in this file still?

Reply:
still trying to resolve the falling with existing integration tests first, which may affect the unit test.

Reply:
unit test is now included